client/src/main/java/io/spine/client/PostingErrorHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.client;
+
+import com.google.protobuf.Message;
+import io.spine.base.Error;
+
+import java.util.function.BiConsumer;
+
+/**
+ * A handler for an error occurred when posting a message for asynchronous processing
+ * at the server side.
+ *
+ * <p>The handler accepts a message which caused an error, and an {@link io.spine.base.Error Error}
+ * obtained from the server.
+ */
+public interface PostingErrorHandler extends BiConsumer<Message, Error> {
```

Comment:
`Posting` tells that this error handler posts something. I suggest `MessageErrorHandler` or `ServerErrorHandler`.

Reply:
That's a difficult one. The problem with the `Server` prefix is that in combination with `Handler` it sounds as if we can handle the error that happened at the server-side.

Reply:
Don't we?

Reply:
Agreed. Going to rename it.